lightning/src/offers/invoice_macros.rs

```diff
@@ -87,6 +87,14 @@ macro_rules! invoice_builder_methods_test { (
 	$self: ident, $self_type: ty, $invoice_fields: expr, $return_type: ty, $return_value: expr
 	$(, $self_mut: tt)?
 ) => {
+	#[cfg_attr(c_bindings, allow(dead_code))]
+	pub(crate) fn amount_msats_unchecked(
```

Comment:
Ah, just realized this breaks CI because it doesn't work for `--cfg=async_payments` unfortunately. Happy to look into a fix if that helps